playbook/run_consumer.go

```diff
@@ -185,16 +192,14 @@ func RunConsumer(c postq.Context) (int, error) {
 		SELECT playbook_runs.*
 		FROM playbook_runs
 		INNER JOIN playbooks ON playbooks.id = playbook_runs.playbook_id
-		WHERE status = ?
```

Comment:
This fixes a bug where a playbook with only agent runners would never be marked as completed